R/target_market_share.R

```diff
@@ -233,8 +233,7 @@ target_market_share <- function(data,
         green_or_brown = "green_or_brown"
       )
     ) %>%
-    warn_if_has_zero_rows("Joining `r2dii.data::green_or_brown` outputs 0 rows") %>%
-    select(-.data$target_name, -.data$green_or_brown)
+    warn_if_has_zero_rows("Joining `r2dii.data::green_or_brown` outputs 0 rows")
```

Comment:
seems a bit strange to throw a warning here, but then act on the situation (return an empty data frame) later

Reply:
Ya this is a good point, going to think if there is a better way to do this so the actions are closer together.